storage/kv.go

```diff
@@ -69,6 +69,9 @@ type KV interface {
 	// Snapshot snapshots the full KV store.
 	Snapshot() Snapshot
 
+	// ForceCommit commits txns into the underlying backend.
+	ForceCommit()
```

Comment:
I wont call this forceCommit here. From external view, this is just a Commit. It ensures all the previously finished operations are persisted to disk.